scthumbd.js

```diff
@@ -79,7 +104,15 @@ if (cluster.isPrimary && workers > 1) {
   app.get('/thumb/*', thumber.thumbnail);
   app.get('/optim/*', thumber.optimize);
 
-  app.listen(port);
+  const server = app.listen(port);
+
+  // When not running in cluster mode, gracefulShutdown uses its own signal handler.
+  const closeHttp = gracefulShutdown(server);
+  process.on('message', (msg) => {
+    if (msg === SHUTDOWN_SIGNAL) {
+      closeHttp().then(() => process.exit());
+    }
+  });
```

Comment:
(related to the other one. also don't forget to remove the unused constant)
```suggestion
  gracefulShutdown(app.listen(port));
```